README.md

```diff
@@ -22,11 +29,6 @@ See test results for your code changes before committing.
 #### Stepback on Failure
 Automatically run past commits to pinpoint the origin of a test failure.
 
-See [the documentation](https://github.com/evergreen-ci/evergreen/wiki) for a full feature list!
-
-## System Requirements
-The Evergreen agent, server, and CLI are supported on Linux, macOS, and Windows.
```

Comment:
I removed this because I believe it duplicates the "Multiplatform Support" section.